packages/jest-jasmine2/src/reporter.ts

```diff
@@ -115,13 +115,9 @@ export default class Jasmine2Reporter implements Reporter {
   private _addMissingMessageToStack(stack: string, message?: string) {
     // Some errors (e.g. Angular injection error) don't prepend error.message
     // to stack, instead the first line of the stack is just plain 'Error'
-    const ERROR_REGEX = /^Error\s*\n/;
-    if (
-      stack &&
-      message &&
-      ERROR_REGEX.test(stack) &&
-      stack.indexOf(message) === -1
-    ) {
+    const ERROR_REGEX = /^Error:?\s\n*/;
```

Comment:
added `:?`

Reply:
Is the change from `\s*\n` -> `\s\n*` also intended?

Reply:
eagle eyes!